libtiff/tif_config.h

```diff
@@ -3,7 +3,7 @@
     hardcoded definitions for MSVC, Xcode and MinGW when using makefiles.
  */
 
-#ifdef __APPLE__
+#if defined(__APPLE__) || defined(__linux__)
```

Comment:
At this stage, should we test for `!defined(_WIN32)` instead perhaps?

Reply:
Yes. I could also swap the cases around and test `#ifdef _WIN32`. But the diff will be bigger. Do you object?

Reply:
No, I think we should avoid handling Linux specifically without any real need, other Unices still exist (albeit barely)...